### DISCUSSION TITLE
Integrate AC70

### DISCUSSION THREAD

**Discussion by tantox1** [Patrick762/hassio-bluetti-bt#134] [Q&A]
Hi, I have some porblem to integrate my new Ac70. It seems all proper installed but I'm unable to see any data from the sensor. 
Any suggestion? thanks
![Ac70](https://github.com/user-attachments/assets/20e13573-3319-4708-8182-44fd06e73fe3)

**Comment by gkathan** [comment]
exactly dame behaviour here with my new AC70 ... anyone any hints ?

* interestingly there is no reload Integration in my side
* I activated debug log level but have no clue where to find those logs 

cheerz

**Comment by AndreaCarriu** [comment]
Hy, i'm in the same situasion with myAC70. Sensor are aviable but no data collected.
here we are my SW version on the APP
IOT v9046.01
ARM v2151.03
DSP v2106.09
i've start supporting the guy and have the time to make some test.

**Comment by tantox1** [comment]
Good morning everyone,
I think they have found a solution here, but I’m not very skilled in IT or Python.
Is there anyone more experienced who could get it working?
[https://github.com/Patrick762/hassio-bluetti-bt/issues/120]
